Frontend/src/Components/Motor_Motion/Motor_Motion.js

```diff
@@ -0,0 +1,448 @@
+    const redTxtCol = getColor("txtRedBg", colorMode);
+    const headerBg = getColor("header", colorMode);
+
+    const fitType = "scale-down";
+
+    const Images = images[`${colorMode}`];
+    
+    return (
+        <Flex flex='auto' direction='column'>
+            <HeadingCell fontSize='2.2vh' label='Motor / Motion'/>
+            <Grid 
+                flex='2'
+                templateRows="1fr 1fr 1fr 1.4fr 1.4fr" 
+                templateColumns="1fr 1fr 4fr 2fr"
+                h='25%'
+                pt='1'
+                pb='1'
+            >
+
+                <GridItem
+                    rowStart={1}
+                    rowSpan={2}
+                    colStart={1}
+                >
+                    <VStack spacing='0px'>
+                        <Text fontSize='1.20vh'>F/R</Text>
+                        <Center h='2.6vh' w='2.6vh' borderWidth='2px' borderRadius='md' textAlign='center'>
+                            <Text as='b' fontSize='1.6vh'>{props.data?.fr_out[0] ? 'T' : 'F'}</Text>
+                        </Center>
+                    </VStack>
+                </GridItem>
+                <GridItem
+                    rowStart={1}
+                    rowSpan={2}
+                    colStart={2}
+                >
+                    <VStack spacing='0px'>
+                        <Text fontSize='1.20vh'>State</Text>
+                        <Center h='2.6vh' w='2.6vh' borderWidth='2px' borderRadius='md' textAlign='center'>
+                            <Text as='b' fontSize='1.6vh'>{props.data?.state[0] ?? "?"}</Text>
+                        </Center>
+                    </VStack>
+                </GridItem>
+
+                <GridItem
+                    rowStart={4}
+                    colStart={1}
+                    colSpan={2}
+                    pl='1'
+                    pr='1'
+                >
+                    <Center
+                        bg={props.data?.motor_controller_contactor[0] ? greenBgCol : redBgCol}
+                        borderColor={borderCol}
+                        borderWidth='2px'
+                        lineHeight='1.6em'
+                        h='71.4%'
+                    >
+                        <Text as='b' fontSize='xs' color={props.data?.motor_controller_contactor[0] ? greenTxtCol : redTxtCol}>Motor {props.data?.motor_controller_contactor[0] ? 'On' : 'Off'}</Text>
+                    </Center>
+                </GridItem>
+
+                <GridItem
+                    rowStart={1}
+                    colStart={3}
+                    borderTop='1px'
+                    borderLeft='1px'
+                    borderRight='1px'
+                >
+                    <Center direction='row' pl='1' pr='1' h='100%'>
+                        <Text fontSize='xs'>Accelerator</Text>
+                        <Spacer/>
+                        <Text fontSize='xs'>{props.data?.accelerator[0].toFixed(0) ?? -1.0} {CONSTANTS.accelerator.UNIT}</Text>
```

Comment:
```suggestion
                        <Text fontSize='sm'>Accelerator</Text>
                        <Spacer/>
                        <Text fontSize='sm'>{props.data?.accelerator[0].toFixed(0) ?? -1.0} {CONSTANTS.accelerator.UNIT}</Text>
```
Let's increase the font sizes in the top middle grid. `xs` looks a little too small